src/lib/components/02-organisms/CategorizedListRoot/CategorizedList/index.js

```diff
@@ -283,7 +346,16 @@ const CategorizedList = React.memo(
                     parentChangeObj,
                     parentComponent
                   );
+                  /**
+                   * The meaning of propsObj has been hard to understand among
+                   * the developers. The object contains values of the original
```

Comment:
Could you explain WHY? Code only says how " Combines the properties of the component with the properties of the change object."